maquetado/scss/details.scss

```diff
@@ -98,27 +110,79 @@
 
 @media only screen and (max-width: 1024px) {
   .book-card {
-    flex-flow: column;
+    box-shadow: 4px 4px 10px $black-russian;
+    display: grid;
```

Comment:
Remember that it is not necessary to repeat the same attributes that appear in desktop. Check these classes

Reply:
Done!